pages/getting-started/first-steps-with-docker.mdx

```diff
@@ -0,0 +1,522 @@
+
+{<h3>Configure installation settings</h3>}
+
+During installation:
+- Opt for WSL 2 based installation instead of Hyper-V.
+- Choose to add a desktop shortcut for easier access.
+
+![Installer Options](/pages/getting-started/first-steps-with-docker/first-steps-with-docker-install.png)
+
+{<h3>Launch Docker Desktop</h3>}
+
+After installation, you can open Docker Desktop by clicking on the newly created icon on your desktop or by searching for it in the Start menu.
+
+{<h3>Accept Docker’s Terms and Conditions</h3>}
+
+Prior to using Docker Desktop, go through the terms and conditions. If you agree, proceed by accepting them.
+
+![Terms and Conditions](/pages/getting-started/first-steps-with-docker/first-steps-with-docker-terms.png)
+
+{<h3>Determine your usage scenario</h3>}
+
+Docker will prompt questions about your intended usage. Answer accordingly to tailor Docker's settings and recommendations to your needs.
+
+{<h3>Post-installation Steps</h3>}
+
+After successful installation, you'll be presented with Docker Desktop’s user interface.
+
+![Docker Desktop UI](/pages/getting-started/first-steps-with-docker/first-steps-with-docker-desktop.png)
+
+Navigate to Docker's settings. Based on your specified usage scenario, verify if Docker is set to start on system boot and adjust this settings.
+
+![Docker Settings](/pages/getting-started/first-steps-with-docker/first-steps-with-docker-settings.png)
+
+<Callout type="info">
+If Docker Desktop is configured not to launch on system startup, you will have to start it manually every time you wish to use it.
+</Callout>
+
+</Steps>
+
+## Verifying Docker Installation
+
+To ensure that Docker has been successfully installed and is running on your system, follow the steps below:
+
+<Steps>
+{<h3>Open the Command Line or Terminal</h3>}
+
+- **Windows**: Press `Windows + R`, type `cmd`, and hit Enter.
+- **Mac**: Press `Command + Space`, type `terminal`, and hit Enter.
+- **Linux**: Depending on the distribution, you can use shortcuts like `Ctrl + Alt + T` or access the terminal from the application menu.
+
+{<h3>Verify Docker Version</h3>}
+
+Type the following command:
+
+```
+docker --version
+```
+
+This should display the version of Docker installed on your machine. For example:
+
+```
+Docker version 24.0.6, build ed223bc
+```
+
+{<h3>Check Docker Details</h3>}
+
+For more detailed information about the Docker installation, you can use:
+
+```
+docker info
+```
+
+This command provides a comprehensive overview of your Docker setup, including the number of containers running, paused, and stopped, as well as other system information.
+
+{<h3>Ensure Docker Daemon is Running</h3>}
+
+To check that the Docker daemon (background service) is running, execute:
+
+```
+docker system info
+```
+
+If Docker is running correctly, this command will return system-wide information. If not, you may receive an error indicating that Docker is not running.
```

Comment:
Maybe an image of what you should expect or C/P from the console? just a suggestion

Reply:
The output is very informative, but it is to long so I would skip this one.